mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/RabbitMQEventBus.java

```diff
@@ -124,4 +124,15 @@ public Registration register(MailboxListener listener, Group group) {
         }
         throw new IllegalStateException("Event Bus is not running");
     }
+
+    @Override
+    public Mono<Void> reDeliver(Group group, Event event) {
+        if (isRunning) {
```

Comment:
Maybe this should be tested by a Preconditions isn't it?

And let's add this precondition everywhere in this class BTW...